README.md

```diff
@@ -28,10 +32,9 @@ openssl rsa -in carta_private.pem -outform PEM -pubout -out carta_public.pem
 ```
 
 ## Server Configuration
+Server configuration is handled by a configuration file in JSON format, adhering to the [CARTA config schema](config/config_schema.json). Additional details can be found in the auto-generated [config documentation](docs/config_schema.html) or the [example config](config/example_config.json). By default, the server assumes the config file is located at `/etc/carta/config.json`, but this can be changed by using the `--config` or `-c` command line argument when running the server. 
```

Comment:
"this can be changed by using" --> "you can change this with"

Reply:
> It would still be good to move to a single script being called by sudoers, but this is good for the configuration update

agreed. That can be done in a separate PR I guess.